### ISSUE TITLE
Uncaught SyntaxError: Strict mode code may not include a with statement

### ISSUE THREAD

**Issue by hmontes** [uploadcare/uploadcare-widget-react-demo#1]
Hi

I'm installed the uploadcare widget (npm install --save uploadcare-widget) and i put this code:

```
import React from 'react';
import uploadcare from 'uploadcare-widget';

class ImageUpload extends React.Component {

  componentDidMount() {
    uploadcare.start({
      publicKey: "demopublickey",
      tabs: "all"
    });
  }

  render() {
    return (
      <div>
        <input type="text" role="uploadcare-uploader"/>
      </div>
    )
  }
}

export default ImageUpload;
```

But i obtain this error:

```
uploadcare.min.js:12 Uncaught SyntaxError: Strict mode code may not include a with statement
```

What is the problem? Thanks

BTW. I don't use Webpack, instead, i use the awesome brunch.io

**Comment by Zmoki** [comment]
Hi @hmontes! Thank you for issue =) I will try to use brunch and reproduce your error and then come back to you

**Comment by Zmoki** [comment]
@hmontes can you share your brunch config?

**Comment by hmontes** [comment]
Yeah. Sure!

```
// See http://brunch.io for documentation.
exports.config = {
  hot: true,

  files: {
    javascripts: {joinTo: 'app.js'},
    stylesheets: {joinTo: 'app.css'},
  },

  plugins: {
    babel: {
      presets: ['react', 'es2015', 'es2016', 'stage-0']
    }
  },
};
```

**Comment by Zmoki** [comment]
The widget use `with` function that not support in strict mode.

Brunch by default don't switch on [strict mode for third party](https://github.com/brunch/brunch-guide/blob/master/content/en/chapter03-conventions-and-defaults.md#commonjs-module-wrapping), but in your case strict mode is on for some reason. Maybe with `hot: true` it happens.

I created brunch project from [with-react](https://github.com/brunch/with-react) - https://github.com/Zmoki/try-brunch-react
I added uploadcare widget and all works fine without hmr.

I tried to use `hot: true`, but I get error `warn: HMR runtime not found for public/app.js.` from brunch and I'm stuck on this =))

Try to off hmr. Your error occurs with `hot: false`?

@hmontes
